packages/platforms/quanta/x86-64/x86-64-quanta-ly8-rangeley/onlp/builds/src/x86_64_quanta_ly8_rangeley/module/src/sysi.c

```diff
@@ -76,3 +90,62 @@ onlp_sysi_oids_get(onlp_oid_t* table, int max)
      */
     return 0;
 }
+
+int
+onlp_sysi_platform_manage_leds(void)
+{
+    led_control.PMCnt++;
+    if(led_control.PMCnt>300)
+        led_control.PMCnt = 0;
+    if(led_control.PMCnt % 5 == 1){//Each 10 seconds detect one time
+        if(led_control.psu1_mvin != 0) {
+            pca953x_gpio_value_set(PSU_GPIO_PSU1_GREEN_R, 1);
```

Comment:
@jonathantsai-qci These calls to change the LEDs should be made using the public onlp_led APIs rather than the lower-level implementation.